src/editors/playgroundController.ts

```diff
@@ -572,11 +586,18 @@ export default class PlaygroundController {
     }
   }
 
-  async evaluateParticipantQuery(text: string): Promise<boolean> {
+  async evaluateParticipantCode(codeToEvaluate: string): Promise<boolean> {
     const shouldConfirmRunCopilotCode = vscode.workspace
       .getConfiguration('mdb')
       .get('confirmRunCopilotCode');
 
+    if (!this._connectionController.isCurrentlyConnected()) {
+      // TODO(VSCODE-618): Prompt user to connect when clicked.
+      void vscode.window.showErrorMessage(connectBeforeRunningMessage);
```

Comment:
Another drive-by, adding an error message when trying to run when not connected, and a todo for the changes in VSCODE-618.